keyserver/README.md

```diff
@@ -0,0 +1,35 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+```
+⇒  go install github.com/coniks-sys/coniks-go/keyserver/coniksserver
+⇒  coniksserver -h
+Usage of coniksserver:
+  -config string
+    	Path to the configuration file (default "config.toml")
+  -dir string
+    	Directory location for storing generated files (default ".")
+  -genconfig
+    	Generate server configurations file
+  -genkey
+    	Generate signing/VRF key, e.g. -genkey=sign,vrf
+```
+Generate a private key for secure connection (TLS)
+```
+⇒  openssl ecparam -genkey -name prime256v1 -out server.key
+```
+
+Generation of self-signed(x509) public key (PEM-encodings `.pem`) based on the private (`.key`)
+```
+⇒  openssl req -new -x509 -sha256 -key server.key -out server.pem -days 3650
+```
+
+Run the server
+```
+⇒  mkdir coniks/
+⇒  coniksserver -dir coniks -genkey -genconfig
+⇒  coniksserver -dir coniks  // run
```

Comment:
Cool, that was quick. [cobra](https://github.com/spf13/cobra) looks indeed very nice. I'm also slightly more familiar with [cli](https://github.com/urfave/cli). Never compared both. BUt I guess any of those equally suit our needs.